packages/server/src/schema/uuid/taxonomy-term/resolvers.ts

```diff
@@ -104,59 +109,68 @@ export const resolvers: TypeResolvers<TaxonomyTerm> &
     },
   },
   TaxonomyTermMutation: {
-    async setNameAndDescription(_parent, { input }, { dataSources, userId }) {
+    async create(_parent, { input }, { dataSources, userId }) {
       assertUserIsAuthenticated(userId)
 
-      const { id, name, description = null } = input
+      const { parentId, name, taxonomyType, description = null } = input
 
       assertArgumentIsNotEmpty({ name })
 
       const scope = await fetchScopeOfUuid({
-        id,
+        id: parentId,
         dataSources,
       })
 
       await assertUserIsAuthorized({
         userId,
         dataSources,
-        message:
-          'You are not allowed to set name or description of this taxonomy term.',
-        guard: serloAuth.TaxonomyTerm.set(scope),
+        message: 'You are not allowed create taxonomy terms.',
+        guard: serloAuth.Uuid.create('TaxonomyTerm')(scope),
       })
 
-      const { success } =
-        await dataSources.model.serlo.setTaxonomyTermNameAndDescription({
-          id,
-          name,
-          description,
-          userId,
-        })
+      const parent = await assertIsTaxonomyTerm(parentId, dataSources)
```

Comment:
```suggestion
      const parent = await getTaxonomyTerm(parentId, dataSources)
```

Here we need two functions since I would never accept that a `assert` function returns a value...